src/ui/ons/ONSAutosuggest.svelte

```diff
@@ -46,7 +46,7 @@
           on:keyup={onKeyUp}
           class="ons-input ons-input--text ons-input-type__input ons-js-autosuggest-input "
           autocomplete="off"
-          aria-describedby="{id}-label-description-hint"
+          aria-describedby={hint ? `${id}-label-description-hint` : ""}
```

Comment:
FYI: We should probably make the whole attribute conditional, but [it's still fine as is](https://www.tpgi.com/describing-aria-describedby/#comment-1060):
> this latter approach did cause some screen readers to pause very briefly as they attempted to read the empty description.